### ISSUE TITLE
Unable to clone your repository using your documentation

### ISSUE THREAD

**Issue by berdal84** [emilk/imgui_software_renderer#1]
I'm unable to clone your repo using `git clone git@github.com:emilk/imgui_sw.git`
I got an error relative to user rights (not able to clone). 

```
git clone git@github.com:emilk/imgui_sw.git
Clonage dans 'imgui_sw'...
The authenticity of host 'github.com (192.30.253.112)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
Are you sure you want to continue connecting (yes/no)? yes
Warning: Permanently added 'github.com,192.30.253.112' (RSA) to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.
Please make sure you have the correct access rights
and the repository exists.
```

So I suggest to change your documentation line by : 
`git clone https://github.com/emilk/imgui_software_renderer.git`

I'vve an issue with the submodule init too:
```
 git submodule update --init --recursive
Sous-module 'third_party/emath' (git@github.com:emilk/emath.git) enregistré pour le chemin 'third_party/emath'
Sous-module 'third_party/emilib' (git@github.com:emilk/emilib.git) enregistré pour le chemin 'third_party/emilib'
Sous-module 'third_party/imgui' (git@github.com:ocornut/imgui.git) enregistré pour le chemin 'third_party/imgui'
Sous-module 'third_party/stb' (git@github.com:nothings/stb.git) enregistré pour le chemin 'third_party/stb'
Clonage dans 'third_party/emath'...
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: Le clonage de 'git@github.com:emilk/emath.git' dans le chemin de sous-module 'third_party/emath' a échoué
```

**Comment by emilk** [comment]
I've updated the documentation, thanks for pointing that out!

As for the second problem:

Does `git clone git@github.com:ocornut/imgui.git` work for you?
Does `git clone git@github.com:emilk/emilib.git` work for you?
Does `git clone git@github.com:emilk/emath.git` work for you?

**Comment by berdal84** [comment]
Thanks for your answer emilk. No I can't do any of them. Same access rights problem.

**Comment by emilk** [comment]
So how come `git clone https://github.com/emilk/imgui_software_renderer.git` works for you?

**Comment by emilk** [comment]
Oh, I see – I should change them all to `https`. I will get on that shortly!

**Comment by emilk** [comment]
Fixed in a76294b7f14bdb0d56947cf98d4cb15ff9b1dd57

Thanks @berdal84 !
